addons/modules/stringtable.xml

```diff
@@ -448,111 +487,127 @@
             <Chinese>慢速 (步行)</Chinese>
             <Chinesesimp>慢速 (步行)</Chinesesimp>
             <Russian>Ограниченный (Ходьба)</Russian>
+            <German>Limitiert(Gehen)</German>
```

Comment:
With or without space in front of the `(`?
I would prefer one type through the naming - Ref: Combat modes